### PR TITLE
Adds react drag and drop uploader

### PR DESCRIPTION
**Pull request by dennyluan**
This PR adds a react drag-and-drop component `Uploader` for new uploads. The front end upload system uses Uppy. 

Also notice @cindywu @alexkrolick also doing some housecleaning by removing all generated minitest files (everything found in `/test` folder), since we'll be using RSpec in favor of minitest. So just a heads up in your branches.

### CODE REVIEW

File: app/javascript/components/Uploader.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	 	const data = {
+			file_id: id,
+			upload: {}
+	 	}
+		superagent.post(url)
+			.send(data)
+			.set('X-CSRF-Token', token)
+			.set('accept', 'application/json')
+			.then(res => {
+				console.log(res)
```
Comment (cindywu) [inline app/javascript/components/Uploader.js]: did you intend to keep this console.log in here?

File: app/javascript/components/Uploader.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+				window.location = res.body.redirect_to
+		  })
+		  .catch(err => {
+		  	console.log(err.message)
```
Comment (cindywu) [inline app/javascript/components/Uploader.js]: another console.log. is keeping this here intentional?

File: app/assets/stylesheets/pages.scss [hunk 3]
```diff
@@ -16,16 +16,12 @@ header {
     padding: 50px 0;
 
     .btn-primary {
-      color: $white;
-      background-color: $brandGreen;
-      border-color: transparent;
+      // color: $white;
```
Comment (cindywu) [inline app/assets/stylesheets/pages.scss]: why is it important to preserve this commented out code?

File: app/workers/figures_fetcher_worker.rb [hunk 4]
```diff
@@ -26,12 +26,11 @@ def perform(figures_response, upload_id)
...
-    puts ">>>>>> figures API response"
-    puts figures_response
-    puts "<<<<<<"
+    # calls figures to delete assets
     FiguresExtractService.cleanup(@upload.id)
-    FiguresInlinerService.call(@upload.post.id)
 
+    # TODO: move this step into a batch
+    # run step to inject figures into post body
+    FiguresInlinerService.call(@upload.post.id)
```
Comment (cindywu) [inline app/workers/figures_fetcher_worker.rb]: On first read, I thought ```Inliner``` in ```FiguresInlinerService``` was a typo. I was expecting ```FiguresInlineService```. ```Inliner``` can be hard to remember and easy to mistype. I am n=1.

**Comment by cindywu** [comment]
I don't have any critical objections. 👍🏽

File: app/assets/stylesheets/pages.scss (hunk 3 above)
Comment (dennyluan) [reply app/assets/stylesheets/pages.scss]: am waiting for the new styleguide PR to go in first before I fix the buttons

**Pull request merged**